### PR TITLE
Feature/learn upon integration

### PR DESCRIPTION
**Pull request by Wardormeur**
TODO : init env var for productions

### CODE REVIEW

File: lib/users/lms/check-certificate-authenticity.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+'use strict';
+var async = require('async');
+var _ = require('lodash');
+var crypto = require('crypto');
+var Uuid = require('node-uuid');
```
Comment (DanielBrierton) [inline lib/users/lms/check-certificate-authenticity.js]: Not used

File: lib/users/lms/check-certificate-authenticity.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+'use strict';
+var async = require('async');
```
Comment (DanielBrierton) [inline lib/users/lms/check-certificate-authenticity.js]: Not used

**Comment by Wardormeur** [comment]
sadness overflow to the unused packages :>

File: lib/users/lms/get-lms-link.js [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+   * @param  {Fn} waterfallCb
+   */
+  function getUserMemberships (lmsUserId, userTypes, waterfallCb) {
+    // Get the user in LearnUpon
+    request.get(APIUrl + 'group_memberships?user_id=' + lmsUserId, {
+      auth: {
+        user: LMSUsername,
+        pass: LMSPassword
+      },
+      json: {
```
Comment (DanielBrierton) [inline lib/users/lms/get-lms-link.js]: I don't think this is needed? Same info is in the URL, which is what the docs say to do.

**Comment by DanielBrierton** [comment]
As per my comment on CoderDojo/cp-zen-platform#929, it all works. Just the question on "Other" badges

**Pull request merged**